apps/web/src/components/CTAButton.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+import React from "react";
+import { useRouter } from "next/navigation";
+
+type CTAButtonProps = {
+  text: string;
+  to?: string;
+  href?: string;
+};
+
+export default function CTAButton({ text, to, href }: CTAButtonProps) {
```

Comment:
Why have `to` and `href`? Is it to --> internal page and href --> external page?